api/overview/azure/edgeorder.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Azure Edge Hardware Center libraries for .NET
+description: Reference for Azure Edge Hardware Center libraries for .NET
+ms.date: 02/09/2021
+ms.topic: reference
+ms.service: edgeorder
+---
+# Azure Edge Hardware Center APIs for .NET
+
+## Overview
+
+[Azure Edge Hardware Center](https://docs.microsoft.com/en-us/azure/azure-edge-hardware-center/) is an Azure service that lets you order a variety of first party Azure hardware and discover third party hardware offered by our partners. Edge Hardware Center lets you see and track all your order related information at one place.
```

Comment:
```suggestion
[Azure Edge Hardware Center](https://docs.microsoft.com/azure/azure-edge-hardware-center/) is an Azure service that lets you order a variety of first party Azure hardware and discover third party hardware offered by our partners. Edge Hardware Center lets you see and track all your order related information at one place.
```

Without the culture slug, MS docs will redirect to the version appropriate for the current language set in the browser.